src/main/java/frc/robot/subsystems/Superstructure.java

```diff
@@ -16,5 +17,6 @@ public Superstructure(RollerSubsystem rollerSubsystem, TiltedElevatorSubsystem t
     @Override
     public void periodic() {
         tiltedElevatorSubsystem.pieceGrabbed = rollerSubsystem.getPiece() != HeldPiece.EMPTY;
+        rollerSubsystem.allowOpen = tiltedElevatorSubsystem.getHeight() >= Units.inchesToMeters(20);
```

Comment:
arbitrary, can probably add to constants?

Reply:
thumb up